src/tribler/core/tests/test_search_utils.py

```diff
@@ -15,3 +21,129 @@ def test_filter_keywords():
     result = filter_keywords(["to", "be", "or", "not", "to", "be"])
     assert isinstance(result, list)
     assert len(result) == 4
+
+
+def test_torrent_rank():
+    query = 'Big Buck Bunny'
+
+    # The exact match ranked as pretty high
+
+    r1 = torrent_rank(query, 'Big Buck Bunny')  # 0.81
```

Comment:
Because 1 is supposed to be the highest rank. For rank to be highest, a torrent should have an exact title match and an infinite number of seeders. So, if a torrent has an ideal title match but zero seeders, it still is not good enough to have the rank 1.

Reply:
Thank you for the explanation.